cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -446,7 +406,8 @@ def to_schema(self) -> DMSSchema:
                         if isinstance(prop.value_type, ViewEntity):
                             source = prop.value_type.as_id(default_space, default_version)
                         else:
-                            source = dm.ViewId(default_space, prop.value_type, default_version)
+                            # CRITICAL COMMENT: NOT SURE WHY IS THIS ALLOWED!?
+                            source = dm.ViewId(default_space, prop.value_type.suffix, default_version)
```

Comment:
This is the case when the `value_type` is a string and we know it is a edge/relation. Then, we assume the default space and version. However, based on your new parsing of the value_type it might be impossible to get into this situation.